src/components/app/app.tsx

```diff
@@ -10,22 +10,27 @@ import Layout from '../layout/layout';
 import ScrollToTop from '../scroll-to-top';
 import PrivateRoute from '../private-route/private-route';
 import { getAuthorizationStatus } from '../../mock/getAuthorizationStatus';
+import { TOffer } from '../../types/offers';
+import { TReview } from '../../types/reviews';
```

Comment:
```suggestion
import type { TReview } from '../../types/reviews';
```

Советую типу импортировать вот так, таким образом мы говорим бандлеру, что это тип и не нужно включать его в бандл. Это позволяет уменьшить размер бандла для прод сервера